src/lib.rs

```diff
@@ -95,18 +87,11 @@
 //!
 //! ```
 //! let mut airbrake = airbrake::configure(|config| {
-//!     config.proxy = "127.0.0.1:8080".to_owned();
-//! });
-//! ```
-//!
-//! ### app_version
-//!
-//! The version of your application that you can pass to differentiate errors
-//! between multiple versions. It's not set by default.
```

Comment:
The whole `app_version` section was removed, but shouldn't have been. Need to add it back to document the new form of this feature, which is just the `version` method on the Context